src/controllers/WhoWasIllController.ts

```diff
@@ -0,0 +1,18 @@
+import { SessionMiddleware } from 'ch-node-session-handler';
+import { controller, httpGet } from 'inversify-express-utils';
+
+import { BaseAsyncHttpController } from 'app/controllers/BaseAsyncHttpController';
+import { FeatureToggleMiddleware } from 'app/middleware/FeatureToggleMiddleware';
+import { Feature } from 'app/utils/Feature';
+import { WHO_WAS_ILL_PAGE_URI } from 'app/utils/Paths';
+
+const template = 'illness/who-was-ill';
+
+@controller(WHO_WAS_ILL_PAGE_URI, FeatureToggleMiddleware(Feature.ILLNESS_REASON), SessionMiddleware)
```

Comment:
This feels a bit out of scope, I don't remember from the standup if this is what we agreed to do. Ideally we would have that page ready before implementing this ticket. Can we not link the user to the URL even if a page doesnt exist?

Reply:
We _can_, and it wouldn't affect local unit tests. However, it would lead to a 404 status instead of a 200, which could impact the automated tests. Since you've been drafted into Testing, you'd know about this, so if you feel either a) the tests would be unaffected or b) they _would_ be affected but you'd be happy to update the test once the Who Was Ill page is implemented, then I'm more than happy to get rid of this placeholder page.